doc/introduction.rst

```diff
@@ -201,20 +223,25 @@ same steps as in :ref:`testing_installation`.
 Python for NeuroImaging, a quick start
 ==========================================
 
-If you don't know Python, **Don't panic. Python is easy**. Here, we give
-the basics to help you get started. For a very quick start to the
+If you don't know Python, **Don't panic. Python is easy**. It is important
+to realize that most things you will do in nilearn require only a few or a
+few dozen lines of Python code.
+Here, we give
+the basics to help you get started. For a very quick start into the programming
 language, you can `learn it online <http://www.learnpython.org/>`_.
-For a full blown introduction to
+For a full-blown introduction to
 using Python for science, see the `scipy lecture notes
 <http://scipy-lectures.github.io/>`_.
 
 
 We will be using `IPython <http://ipython.org>`_, which provides an
-interactive scientific environment. Start it with::
+interactive scientific environment that facilitates (e.g., measuring
+execution time of a command) and improves (e.g., printing of large matrices)
```

Comment:
I don't think that measuring the execution time of a command is a useful feature for our users. I think that interactive debugging is more useful.